style.css

```diff
@@ -0,0 +1,1510 @@
+@media screen and (max-width: 768px) {
+  .main-wrap {
+    background-image: url(Images/mobile_intro_bg.png);
+    height: 600px;
+    margin: -7px;
+  }
+
+  .mobile-menu {
+    margin-left: 65px;
+    padding-top: 30px;
+  }
+
+  #menu-over {
+    background-color: #3c3a39;
+    display: none;
+    height: 870px;
+    width: 100%;
+    position: fixed;
+    bottom: 57px;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    mix-blend-mode: normal;
+    box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
+    overflow-x: hidden;
+    transition: 0.75ms;
+  }
+
+  .closeMenu {
+    height: 20px;
+    width: 20px;
+    position: absolute;
+    left: 25%;
+    right: 25%;
+    top: 10px;
+    margin-left: 260px;
+    background-image: url(Images/CancelButton.png);
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .mobile-menu1 > li a {
+    left: 5px;
+    right: 5px;
+    top: calc(50% - 24px / 2);
+    font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Make sure the pages use the same fonts or similar fonts from the original design as the previous reviewer requested. Let's use `Cocogoose`, `Lato` fonts instead of `sans-serif` for every text. Kindly search for `font-family` in your CSS and replace each `sans-serif` with `Cocogoose `or `Lato`. 😃